h/assets.py

```diff
@@ -2,9 +2,41 @@
 import re
 
 from deform.field import Field
+from webassets.filter import ExternalTool, register_filter
 import pyramid
 
 
+class Browserify(ExternalTool):
+    """
+    An input filter for webassets that browserifies CoffeeScript or JavaScript.
+
+    The browserify command-line client has several limitations when piping
```

Comment:
I'd agree, but that's just not how Webassets filters are supposed to work. This is slightly more work but is consistent with the rest of the Webassets universe. Reading from disk is a hack and will bite someone at some point.